tests/Unit/ErrorHandling/Test_SegfaultHandler.cpp

```diff
@@ -0,0 +1,22 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cstdio>
+
+#include "Utilities/ErrorHandling/SegfaultHandler.hpp"
+
+// [[OutputRegex, Segmentation fault!]]
+SPECTRE_TEST_CASE("Unit.ErrorHandling.SegfaultHandler",
+                  "[ErrorHandling][Unit]") {
+  ERROR_TEST();
+  enable_segfault_handler();
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wold-style-cast"
+  // NOLINTNEXTLINE
+  int* foo = (int*)-1;  // make a bad pointer
```

Comment:
Why an old-style cast?

Reply:
This doesn't work reliably anyway. Do you know of a reliable way of triggering a segfault but not a bus error or something like that?

Reply:
Actually nvm, I went with raising the signal manually.